### ISSUE TITLE
[LoRA CI] Test LR scheduler

### ISSUE THREAD

**Issue by rohan-varma** [pytorch/torchtune#379]
- LR scheduler is used in LoRA (as well as an option for full finetune), we should thus ensure it is covered by CI.
